src/scenes/Event/store.js

```diff
@@ -21,6 +21,7 @@ const { BETTING, ORACLE_RESULT_SETTING, OPEN_RESULT_SETTING, ARBITRATION, WITHDR
 const INIT = {
   loading: true,
   event: undefined,
+  betRoundBets: undefined,
```

Comment:
cant you just get this from `event.roundBets`?

Reply:
looks like this is unused, delete if unused. when i refactored the event store i tried the store as clean as possible of an `event` duplicated vars. if its already in `event` and we dont need to alter it, then keep it there and access it from `event.`